graph/patterns/operations.md

```diff
@@ -33,32 +41,34 @@ You can consider related patterns such as [long running operations](./long-runni
 
 - Microsoft Graph supports the use of optional parameters. The optional parameter annotation can be used instead of creating function or action overloads when unnecessary.
 
-- API designer **MUST** use POST to call operations on resources.
+- API designer **MUST** use POST to call actions on resources.
+- API designer **MUST** use GET to call functions on resources.
 
-- The addition of a new mandatory not-nullable parameter to an existing action or function is a breaking change and is not allowed without proper versioning.
+- The addition of a new mandatory not-nullable parameter to an existing action or function is a breaking change and is not allowed without proper versioning that is in accordance with our [deprecation guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/graph/deprecation.md).
 
 ## Example
```

Comment:
I feel like this section is a bit incomplete.
It would be great to add to each of these:

1. A 1-2 sentence scenario explanation, with why it makes sense as a function/action
2. Show the actual request and response to go with the XML
3. Strip out the AGS annotations - they are confusing and not required for defining these functions/actions.

Reply:
@dkershaw10 Do you think this is a good set of examples?